### PR TITLE
Add alignment buttons for embed

### PR DESCRIPTION
**Pull request by ellatrix**
_Shamelessly copied from the image block._ See #729.

### CODE REVIEW

**Comment by jasmussen** [comment]
Love it! Works great! 👍 👍

**Comment by iseulde** [comment]
I notice that the toolbar position is a bit strange when it's "wide", but same for image. Let's address in a separate issue.

**Comment by jasmussen** [comment]
> I notice that the toolbar position is a bit strange when it's "wide", but same for image. Let's address in a separate issue.

This is being addressed largely in https://github.com/WordPress/gutenberg/pull/812

**Review by youknowriad — APPROVED** [review]
Worth nothing that if we do #830 These controls could be factorized.

LGTM 👍

File: blocks/library/image/index.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ registerBlock( 'core/image', {
 		{
 			icon: 'align-center',
 			title: wp.i18n.__( 'Align center' ),
-			isActive: ( { align } ) => 'center' === align,
+			isActive: ( { align } ) => ! align || 'center' === align,
```
Comment (aduth) [inline blocks/library/image/index.js]: No align isn't same as center aligned, is it?

Comment (iseulde) [inline blocks/library/image/index.js]: Maybe I'm confusing this... It has the same style.

Comment (aduth) [inline blocks/library/image/index.js]: It appears the same in the editor, but different on the front-end.

none|center
---|---
none|center

We should probably make this distinction more obvious in the editor though.

Comment (jasmussen) [inline blocks/library/image/index.js]: Probably would be good to get https://github.com/WordPress/gutenberg/pull/812 in, before fixing centering issues.

Comment (iseulde) [inline blocks/library/image/index.js]: I'm fine with reverting that part.

**Pull request merged**